tools/deb_to_rb.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+set -e
+
+CREW_PREFIX=/usr/local
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+pkg_prefix=${CREW_LIB_PREFIX}/crew/packages
+GREEN='\e[1;32m';
+RED='\e[1;31m';
+BLUE='\e[1;34m';
+YELLOW='\e[1;33m';
+RESET='\e[0m'
+
+rm -rf ${out}
+
+rm -rf ${CREW_PREFIX}/tmp/deb
+
+
+cd ${CREW_PREFIX}/tmp/
+
+mkdir -p deb && cd ./deb
+
+echo -e "${YELLOW}Downloading deb...${RESET}"
+curl '-#' ${1} -o deb
+
+ar x deb
+
+
+tar xvf control.tar.*
+  
+name=$(grep "Package: " ./control | sed -e 's/Package: //')
+pkgname=$(echo ${name} | sed -e 's/-/_/g')
+ver=$(grep "Version: " ./control | sed -e 's/Version: //' | sed -e 's/-/_/g')
+info=$(grep "Description: " ./control | sed -e 's/Description: //' | sed -e 's/-/_/g')
+sha256sum=$(sha256sum ./deb | cut -d ' ' -f1)
```

Comment:
Would it make sense to add switch options to add the name, description, homepage, etc.?